client/ayon_maya/plugins/load/load_layout.py

```diff
@@ -0,0 +1,223 @@
+from maya import cmds
+import math
+import json
+import collections
+import ayon_api
+from ayon_maya.api import plugin
+from ayon_maya.api.lib import unique_namespace
+from ayon_core.pipeline import (
+    load_container,
+    get_representation_path,
+    discover_loader_plugins,
+    loaders_from_representation,
+    get_current_project_name
+)
+from maya.api import OpenMaya as om
+from ayon_maya.api.pipeline import containerise
+
+
+class LayoutLoader(plugin.Loader):
+    """Layout Loader (json)"""
+
+    product_types = {"layout"}
+    representations = {"json"}
+
+    label = "Layout Loader"
```

Comment:
As far as I know most "load" entries in are either `Load X`, `Import X` or `Reference X`.
```suggestion
    label = "Load Layout"
```

Generating some consistency in those actions has come up before - e.g. [here](https://github.com/ynput/OpenPype/issues/694#issuecomment-1091081271) but I'm not sure if @mkolar @antirotor has new notes about that.

Reply:
That's my bad. I will just rename it back to the Load Layout